manifest/manifest.go

```diff
@@ -24,6 +24,10 @@ const (
 	DockerV2Schema2ConfigMediaType = "application/vnd.docker.container.image.v1+json"
 	// DockerV2Schema2LayerMediaType is the MIME type used for schema 2 layers.
 	DockerV2Schema2LayerMediaType = "application/vnd.docker.image.rootfs.diff.tar.gzip"
+	// DockerV2Schema2LayerGzipEncMediaType is MIME type used for encrypted compressed layers.
+	DockerV2Schema2LayerGzipEncMediaType = "application/vnd.docker.image.rootfs.diff.tar.gzip+enc"
+	// DockerV2Schema2LayerEncMediaType is MIME type used for encrypted layers.
+	DockerV2Schema2LayerEncMediaType = "application/vnd.docker.image.rootfs.diff.tar+enc"
```

Comment:
Should this be OCI?

Reply:
We only have 2 types of encrypted images. This is similar to what we have in upstream containerd, https://github.com/containerd/containerd/blob/c90a3d49323bf3d9c916afede87337cfafdd9319/images/encryption/encryption.go#L54